DOCS-1285: Document Micro-RDK supported models (#2072)
viamrobotics/docs: docs/micro-rdk/base/_index.md

---
title: "Micro-RDK Base Models"
linkTitle: "Base"
weight: 30
type: "docs"

> description: "Base support in the micro-rdk."

Reviewer: ```suggestion
description: "The micro-RDK support for moving platforms that the other parts of a mobile robot attach to."
```

images: ["/icons/components/base.svg"]

> tags: ["base", "components", "micro-rdk"]

Reviewer 2: ```suggestion
tags: ["base", "components", "micro-rdk"]
no_list: true
```
[opt] I think on all of these pages the cards at the bottom are redundant with the model table so would be good to remove? Marking as opt in case this was intentional

Author: Done!

---
title: "Micro-RDK Base Models"
linkTitle: "Base"
weight: 30
type: "docs"
description: "The micro-RDK support for moving platforms that the other parts of a mobile robot attach to."
images: ["/icons/components/base.svg"]
tags: ["base", "components", "micro-rdk"]
no_list: true
# SMEs: Nick M., Gautham V.
---

> 

Reviewer 2: [opt] I just realized this now so I realize it's late to bring up and could add in a later PR. But I'm wondering--should each of these component pages link to the corresponding RDK component pages for more info on what a base etc. is? Or at least have some information? For example
```suggestion

A base is the moving platform that other parts of a robot attach to.

By configuring a base component, organizing individual components to produce coordinated movement, you gain an interface to control the movement of the whole physical base of the robot without needing to send separate commands to individual motors.

```

Author: @JessamyT Thank you, that is a really good suggestion! Added abridged intros and linking to main docs for each component, LMK what you think.

---
title: "Micro-RDK Base Models"
linkTitle: "Base"
weight: 30
type: "docs"
description: "The micro-RDK support for moving platforms that the other parts of a mobile robot attach to."
images: ["/icons/components/base.svg"]
tags: ["base", "components", "micro-rdk"]
no_list: true
# SMEs: Nick M., Gautham V.
---

A base is the moving platform that the other parts of a mobile robot attach to.

> For more information and models supported by [the RDK (features provided by the full version of `viam-server`)](/internals/RDK/), see the [Base component](/components/base/).

Author: ```suggestion
For more information and models supported by [the RDK (features provided by the full version of `viam-server`)](/internals/rdk/), see the [Base component](/components/base/).
```

> For more information and models supported by [the RDK (features provided by the full version of `viam-server`)](/internals/rdk/), see the [Base component](/components/base/).

Reviewer 2: ```suggestion
For more information and models supported by [the RDK (the full version of `viam-server`)](/internals/rdk/), see [Base Component](/components/base/).
```
[opt] Nice. I think the link text can be shorter on these. And then (nit) I think either capitalize to match the literal title of the doc, or don't capitalize "base" (/other component name)?

Reviewer: I think using the glossary entry will make this a bit prettier.
```suggestion
For more information and models supported by the {{< glossary_tooltip term_id="rdk" text="rdk" >}}, see the [Base Component](/components/base/).
```
